### ISSUE TITLE
Format Subpage | Discussing Manual References Within Learning Content

### ISSUE THREAD

**Issue by lukebelcher** [compensationServiceTraining/curriculumCreationGuide#40]
With the M21-1 reorganization, we learned very quickly how difficult it is to update a course when manual references are discussed either in the audio or video files of the course. My suggestion is include instructions to use broad terms when discussing the manual references rather than citing the reference itself in audio/video.  For example, instead of saying "M21-1 I.i.4. states...", say "The manual reference provided on the screen states.." and provide a hyperlink on the slide.  This will help ensure the most difficult portions of the course to edit don't need to be updated in the case that a reference changes.

**Comment by jrvictor** [comment]
Agreed - I believe it is now a known best practice amongst narrators to avoid reading entire references out due to this very issue. It also adds considerable and unnecessary length to the slide audio files and closed captioning, which is both boring and cumbersome to the listener and makes the transcript document contain redundant information. It also prevents the need to re-record all of the audio files in a presentation so that we don't have inconsistent audio files/narrators. Updating a manual reference on a slide is not difficult to do. Re-recording, syncing and captioning all the audio files however is very time-consuming and cumbersome.

This shouldn't just circulate as a known best practice, it needs to be stated in the CCG. Perhaps there could be a section added discussing narration and audio content where this is included along with other tips/best practices.

**Comment by lllotts** [comment]
I agree to an extent. If the lesson is one that is used consistently in the training curriculum, yes, simply stating "The M21-1 states...." should suffice. This way, as mentioned, when the reference is updated, audio does not need to be updated.

However, when it is a lesson that is a "one and done," or an FY mandated training (e.g., Intro to VASRD updates, GW particulate matter, errors on MST or ALS), those classes that target a specific audience for a specific time period, stating the entire reference in audio is suffice, should the author choose to do it that way. Those lessons are not intended to be used year after year, but rather for a specific period, so stating what the reference is, at that point in time, I believe is fine.

From: Joanne Victor ***@***.***>
Sent: Wednesday, January 12, 2022 6:44 AM
To: compensationServiceTraining/curriculumCreationGuide ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] Re: [compensationServiceTraining/curriculumCreationGuide] Format Subpage | Discussing Manual References Within Learning Content (Issue #40)


Agreed - I believe it is now a known best practice amongst narrators to avoid reading entire references out due to this very issue. It also adds considerable and unnecessary length to the slide audio files and closed captioning, which is both boring and cumbersome to the listener and makes the transcript document contain redundant information. It also prevents the need to re-record all of the audio files in a presentation so that we don't have inconsistent audio files/narrators. Updating a manual reference on a slide is not difficult to do. Re-recording, syncing and captioning all the audio files however is very time-consuming and cumbersome.

This shouldn't just circulate as a known best practice, it needs to be stated in the CCG. Perhaps there could be a section added discussing narration and audio content where this is included along with other tips/best practices.

-
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FcompensationServiceTraining%2FcurriculumCreationGuide%2Fissues%2F40%23issuecomment-1011059460&data=04%7C01%7C%7C578f669cca8046dae3a708d9d5d19602%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637775918408317498%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=ItTorEVDS%2BxSK2h2tln3m%2FxPXOoDCJa7smyuK2wfX1w%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FATIMNIGJN3CM5QNN2GFMLLTUVWARDANCNFSM5LXHJ23A&data=04%7C01%7C%7C578f669cca8046dae3a708d9d5d19602%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637775918408317498%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=trkkzdHzgIni7vnEFrFPdzSXJ6lnY88tR7Fln52Um2s%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C578f669cca8046dae3a708d9d5d19602%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637775918408317498%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=sRBgsJgliHiZl29AH4cNsw5Z%2FI0FE6iWuGJKb70OdQo%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C578f669cca8046dae3a708d9d5d19602%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637775918408317498%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=lbChpzttU2wXwgEVwdnnTfcyreORW49OJwizS3h6LPo%3D&reserved=0>.

**Comment by artlessdodger** [comment]
Just adding in here — to save time when you're updating/adding references, you can (and probably should) maintain documentation of all references (to include associated hyperlinks) you plan on using as part of your project folder. That way, whenever you use one, it's a simple copy/paste from your .txt file (or several pastes from your clipboard history, if you're using multiple references in a slide). Here's an example of how I formatted mine for the MST training update:

![image](https://user-images.githubusercontent.com/51389101/149380368-34d16e95-6dce-4d74-aab2-d466867e05b2.png)

As you can see, that's part of the project folder now and can be used indefinitely by anyone who takes it up later. Because the hyperlinks and labels are plaintext, then can be copied and pasted from the clipboard (or clipboard history), without any merged formatting issues.

If you're unsure on how to use the clipboard history, demonstrations in Office Hours ([February 26, 2021](https://vbatraining.adobeconnect.com/psvg9r385o8l/), and [March 26, 2021](https://vbatraining.adobeconnect.com/p6f90wxozr0a/)) cover this.

**Comment by lukebelcher** [comment]
In an effort to incorporate this recommendation into the CCG, I suggest rewriting the first few paragraphs of the "Reference Formatting" section (Format Subpage) to the following:

REFERENCE FORMATTING

The following lays out a format and order for references on the final "Would you like to know more?" slide(s), as well as other incidental references throughout published files (including audio/video presentations), slide notes, and transcript documents.

Formatting of references is intended to remove any formality barriers between the learner and the link — i.e., make clicking as desirable as possible. References should be hyperlinked in both the final CPTX file and the Student Slide Notes PDF. See the templates section for more details.

As a general rule, avoid sub-paragraph regulatory citations and procedural references beneath the section level on the "Would you like to know more?" slide — such references should be included in the presentation itself where they are contextually relevant. 

Also, avoid listing specific references in audio or video presentations.  Instead, point the learner to a reference on the slide — e.g., “The manual reference listed on this page...”.  Doing this will make it easier to edit the course if the reference changes in the future.

**Note —** Reference order on the "Would you like to know more?" slide should mirror the order presented below.

**Comment by kdavisva** [comment]
There has been a lot of discussion on manual references, where they should be in the captivate module, slide notes, etc. This is my input.
1.	I agree that when discussing manual references in an audio/video, the speaker should not read the entire manual reference. They can say “The manual reference provided on the screen states” or “The screen shows the M21-1 Adjudication Procedures Manual references” or “The screen shows the 38 CFR regulations and the Adjudication Procedures Manual references” or something to that fact. This would make it easier if (and when) the manual references change.
2.	I disagree with end notes. That just adds another document that needs to be updated within all the other documents. I don’t think we need to add additional documents.
3.	While Andrew makes a point about using clipboard notes, most people do not know how to use this feature. 
4.	I can agree with the updated REFERENCE FORMATTING on the “Would you like to know more?” slide – IF and I say IF, you use this slide. There are certain projects that don’t use a “Would you like to know more?” slide and instead has a slide after the objective slide that lists the references. There is nothing in this document that says that is not acceptable. What is needed it make sure references are listed within the module. Where they are placed should be up to the Chief/Project lead of that project. The consistency is having the references listed in the module. 

I am sure there will be more discussion on this topic.
